### ISSUE TITLE
fluidsynth: error: The maximum playback duration has been reached. Terminating player!

### ISSUE THREAD

**Issue by Snoucher** [k4zmu2a/SpaceCadetPinball#186]
This error happens when the music changes and when refocusing the window. Not sure how I can extend this strange duration, but I'm very bummed that this happens.

**Comment by Snoucher** [comment]
Additional note:

Music changes when you start a new game and started a mission (but only after you end it does fluidsynth whine)

**Comment by Snoucher** [comment]
Hold on, there's a new issue on fluidsynth with the same problem I experience (https://github.com/FluidSynth/fluidsynth/issues/1272).

**Comment by Snoucher** [comment]
Looks lie this is an issue with fluidsynth itself, not Space Cadet, I'm going to close this with the assumption that fluidsynth is responsible for this issue.

**Comment by k4zmu2a** [comment]
I agree, this does look like a FluidSynth issue.

In general, downstream error is almost always the case in situations like this this one.
Because pinball has minimal control over MIDI backend chosen by SDL.
Basically just load/unload and start/stop, all done via SDL abstraction layer.
